packages/odyssey-storybook/src/components/odyssey-labs/AppTile/AppTile.stories.tsx

```diff
@@ -317,3 +326,41 @@ export const CustomContent: StoryObj<AppTileProps> = {
     );
   },
 };
+
+export const Loading: StoryObj<AppTileProps> = {
+  render: function C() {
+    const [isDrawerOpen, setIsDrawerOpen] = useState<boolean>(false);
+
+    return (
+      <>
+        <Drawer
+          isOpen={isDrawerOpen}
+          onClose={() => setIsDrawerOpen(false)}
+          showDividers={false}
+        />
+        <Box sx={{ maxWidth: 262 }}>
```

Comment:
Same with this. We should memoize it too rather than passing an object that gets re-created on each change.

But also, why are we using a `Box` with a magic number fixed-width in Storybook? I'm not sure, by looking through the PR, why this needs to be here. I thought our app tiles had max widths?

Reply:
The `AppTile` follows the `Card` approach in not providing a fixed width — it'll expand to occupy the full available width.

The `262px` value here is the current width of the app tiles in End User, but we're leaving the component flexible so they can put it into a grid container of some sort.